src/sections/shared/hierarchy/BreadcrumbsGenerator.tsx

```diff
@@ -81,14 +84,15 @@ const LinkToDvObject = ({
   version
 }: {
   name: string
-  type: string
+  type: DvObjectType
```

Comment:
small thing, since `type` is now DvObjectType, the above  const be  `dvObjectTypeToRoute: Record<DvObjectType, Route>`?

Reply:
Added! Thanks!